src/main/java/se/datasektionen/calypso/models/entities/Item.java

```diff
@@ -152,4 +152,21 @@ public void triggerUpdated() {
 		this.updated = LocalDateTime.now();
 	}
 
+	public Item duplicate() {
+		Item duplicatedItem = new Item();
+		duplicatedItem.titleSwedish = this.titleSwedish;
+		duplicatedItem.contentSwedish = this.contentSwedish;
+		duplicatedItem.titleEnglish = this.titleEnglish;
+		duplicatedItem.contentEnglish = this.contentEnglish;
+		duplicatedItem.itemType = this.itemType;
+		duplicatedItem.sensitive = this.sensitive;
+
+		if (duplicatedItem.itemType == ItemType.EVENT) {
+			duplicatedItem.eventLocation = this.eventLocation;
+			duplicatedItem.eventStartTime = this.eventStartTime.plusDays(7);
```

Comment:
Could be worth considering a different default time, makes sense for e.g. idrottsnämnden and other weekly events. 

Perhaps better to do do based on the current date, rather than the event time though, so that you can duplicate old events as well.

But I don't think that it will be a practical problem, just a few more clicks sometimes.

Reply:
I was thinking of keeping the time but clearing the date field. But was not sure how to do it.